app/Models/HttpResponseCode.php

```diff
@@ -0,0 +1,131 @@
+
+	public const MESSAGE_100 = 'Continue';
+	public const MESSAGE_101 = 'Switching Protocols';
+	public const MESSAGE_102 = 'Processing';
+	public const MESSAGE_103 = 'Early Hints';
+	public const MESSAGE_200 = 'OK';
+	public const MESSAGE_201 = 'Created';
+	public const MESSAGE_202 = 'Accepted';
+	public const MESSAGE_203 = 'Non-Authoritative Information';
+	public const MESSAGE_204 = 'No Content';
+	public const MESSAGE_205 = 'Reset Content';
+	public const MESSAGE_206 = 'Partial Content';
+	public const MESSAGE_207 = 'Multi-Status';
+	public const MESSAGE_208 = 'Already Reported';
+	public const MESSAGE_226 = 'IM Used';
+	public const MESSAGE_300 = 'Multiple Choices';
+	public const MESSAGE_301 = 'Moved Permanently';
+	public const MESSAGE_302 = 'Found';
+	public const MESSAGE_303 = 'See Other';
+	public const MESSAGE_304 = 'Not Modified';
+	public const MESSAGE_305 = 'Use Proxy';
+	public const MESSAGE_307 = 'Temporary Redirect';
+	public const MESSAGE_308 = 'Permanent Redirect';
+	public const MESSAGE_400 = 'Bad Request';
+	public const MESSAGE_401 = 'Unauthorized';
+	public const MESSAGE_402 = 'Payment Required';
+	public const MESSAGE_403 = 'Forbidden';
+	public const MESSAGE_404 = 'Not Found';
+	public const MESSAGE_405 = 'Method Not Allowed';
+	public const MESSAGE_406 = 'Not Acceptable';
+	public const MESSAGE_407 = 'Proxy Authentication Required';
+	public const MESSAGE_408 = 'Request Timeout';
+	public const MESSAGE_409 = 'Conflict';
+	public const MESSAGE_410 = 'Gone';
+	public const MESSAGE_411 = 'Length Required';
+	public const MESSAGE_412 = 'Precondition Failed';
+	public const MESSAGE_413 = 'Content Too Large';
+	public const MESSAGE_414 = 'URI Too Long';
+	public const MESSAGE_415 = 'Unsupported Media Type';
+	public const MESSAGE_416 = 'Range Not Satisfiable';
+	public const MESSAGE_417 = 'Expectation Failed';
+	public const MESSAGE_418 = 'I\'m a teapot';
+	public const MESSAGE_421 = 'Misdirected Request';
+	public const MESSAGE_422 = 'Unprocessable Content';
+	public const MESSAGE_423 = 'Locked';
+	public const MESSAGE_424 = 'Failed Dependency';
+	public const MESSAGE_425 = 'Too Early';
+	public const MESSAGE_426 = 'Upgrade Required';
+	public const MESSAGE_428 = 'Precondition Required';
+	public const MESSAGE_429 = 'Too Many Requests';
+	public const MESSAGE_431 = 'Request Header Fields Too Large';
+	public const MESSAGE_451 = 'Unavailable For Legal Reasons';
+	public const MESSAGE_500 = 'Internal Server Error';
+	public const MESSAGE_501 = 'Not Implemented';
+	public const MESSAGE_502 = 'Bad Gateway';
+	public const MESSAGE_503 = 'Service Unavailable';
+	public const MESSAGE_504 = 'Gateway Timeout';
+	public const MESSAGE_505 = 'HTTP Version Not Supported';
+	public const MESSAGE_506 = 'Variant Also Negotiates';
+	public const MESSAGE_507 = 'Insufficient Storage';
+	public const MESSAGE_508 = 'Loop Detected';
+	public const MESSAGE_510 = 'Not Extended';
+	public const MESSAGE_511 = 'Network Authentication Required';
```

Comment:
This messages are unused at all.

Could be used maybe in 16
[app/Controllers/errorController.ph](https://github.com/FreshRSS/FreshRSS/pull/6355/files#diff-b45443632ee435eff9b2f65c74176c014168f72730248209f3701a4ac785f7c8)